### ISSUE TITLE
Use flow-maven-plugin in the Bakery to bundle things

### ISSUE THREAD

**Issue by SomeoneToIgnore** [vaadin/flow-maven-plugin#41]
We need the real application using the plugin bundling features so we can 
* determine what's the performance impact of bundling
* determine missing or oddly working features in the plugin

For now it is enough to have two fragments. One for just the login page needed resources, and one for everything else. It might be nice to know the difference in performance between having just one fragment (the default) compared to the split.

**Comment by denis-anisimov** [comment]
There are a number of issue with the plugin (#45, #46, #47) which blocks this task.
I stop to work on it for now.
It might be that the bakery project is not a good one for this specific task since it's Spring Boot based and it introduces more complexity on its own with plugin configuration because of Spring.

May be it should be split up to two tasks: one for fragments with plain Web project and another task for Spring.

**Comment by pleku** [comment]
We'll need to get the plugin to work in Bakery anyway, so let's fix the plugin for that.

**Comment by denis-anisimov** [comment]
Sure.
Then as a first task the plugin should be fixed so that it works with Bakery and then Bakery should be updated to use the pugin.
After that this task can be taken again.
So the task is blocked by:
- fix maven plugin so that it works with bakery
- update bakery to use the plugin

**Comment by SomeoneToIgnore** [comment]
Issue moved to [vaadin/flow #3319](https://github.com/vaadin/flow/issues/3319) via [**ZenHub**](https://www.zenhub.com/)
